packages/node_modules/@ciscospark/spark-core/test/unit/spec/spark-core.js

```diff
@@ -99,6 +99,85 @@ describe('Spark', () => {
       });
   });
 
+  describe('initializes with Bearer Token', () => {
```

Comment:
This structure could be simplified (because you're verifying the same information) by looping through an array of objects `[{access_token: 'Bearer1234', testName: 'initializes...'}, {access_token: 'Bearer 1234', ...}]`.